cfme/tests/cloud/test_providers.py

```diff
@@ -1256,7 +1256,19 @@ def test_ec2_create_sns_topic():
             1.
             2. Check SNS topic for this region in AWS
```

Comment:
It would be better if this read "Check SNS topic **exists** for this region in AWS". I got confused

Reply:
You are right! Added exists to expectedResults.